features/fixtures/shared/scenarios/OOMScenario.m

```diff
@@ -20,9 +20,14 @@
 
 @implementation OOMScenario
 
+void onCrashHandler(const BSG_KSCrashReportWriter *writer) {
+    assert(!!"onCrashHandler should not be called for OOMs");
```

Comment:
Isn't this always true, so the `assert()` does nothing?

Reply:
You're right, this should be

```suggestion
    assert(!"onCrashHandler should not be called for OOMs");
```

Reply:
Fixed in d2ddf07beb4bf2d1b60a5ac08ffc0776c9c7b7d9